hydra_agent/redis_core/graphutils_operations.py

```diff
@@ -43,10 +45,12 @@ def get_processing(self, url: str, resource: dict) -> list:
         url_list = url_list.split('/')
         # Updating Redis
         # First case - When processing a GET for a resource
+
         if len(url_list) == 3:
             entrypoint, resource_endpoint, resource_id = url_list
-
             # Building the the collection id, i.e. vocab:Entrypoint/Collection
+            # TODO build the correct collection id from the resource
+            # Earlier it used to come as collection/<id> now it comes as class/<id>
             redis_collection_id = self.vocabulary + \
```

Comment:
@xadahiya @chrizandr Earlier when we hit `/DroneCollection/<id>` we used to get one instance of Drone and thus we used to update the DroneCollection member in redis. But according to the latest changes in hydrus,  `/Drone/<id>` returns the instance of the Drone. So how do we know of which collection this class instance will belong to in order to update the redis cache?

Reply:
Drone/<id> should be used for Drone instances. DroneCollection/<id> should be used for collection instances. Collections only contain hydra:Links, so you never update a class instance at the collection. You only update at Drone/<id> to update the drone.